frontend/src/packages/survey/libs/enums/steps.enum.ts

```diff
@@ -0,0 +1,10 @@
+const SurveySteps = {
```

Comment:
enum name should be in single form, also file name should be the same
```suggestion
const SurveyStep = {
```

Reply:
done!